include/toolkitAPI.h

```diff
@@ -570,6 +570,14 @@ int DLLEXPORT swmm_getLinkStats(int index, SM_LinkStats *linkStats);
  @return Error code
 */
 int DLLEXPORT swmm_getPumpStats(int index, SM_PumpStats *pumpStats);
+    
+/**
+@brief Get rainfall information for a gage.
+@param index The index of gage
+@param[out] rainfall and snow associated with the gage.
+@return Error code
+*/
+int DLLEXPORT swmm_getGagePrecip(int index, double *rainfall, double *snowfall, double *total);
```

Comment:
@abhiramm7 

```C
/**
@brief Get rainfall rates for a gage.
@param index The index of gage
@param[out] rainfall rainfall rate
@param[out] snowfall snowfall rate
@param[out] total total precipitation rate
@return Error code
*/
int DLLEXPORT swmm_getGagePrecip(int index, double *rainfall, double *snowfall, double *total);
```

Reply:
We have document generation setup with Doxygen.  It's important to get the doc strings clean:

http://wateranalytics.org/Stormwater-Management-Model/da/d78/group___simulation_settings.html#gaccc662920215c153b43181055c1f3c8f

Reply:
Modified it :) @bemcdonnell